### PR TITLE
Feature/customization

### PR DESCRIPTION
**Pull request by trasher**
This one follows https://github.com/glpi-project/telemetry/pull/11

Only the 3 last commits ("Customization * and fix redirections") has been added. Changes are:
* Move template in subdirectory to be overrided
* Remplace hardcoded project name
* Can disable contact page
* Customize footer links
* Customize logo
* Name routes, use pathfor instead of hardcoded links

### CODE REVIEW

File: PROJECT.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
-Set up a PROJECT
+Set up a project
 ================
 
-By default, the Telemetry app is designed to work along with [GLPI](http://glpi-project.org); but it is popssible to tune a bit the JSON schema in order to fit other projects. This can be achieved using the `config.inc.php` file.
+By default, the Telemetry app is designed to work along with [GLPI](http://glpi-project.org); but it is popssible to tune it a bit in order to fit other projects.
```
Comment (orthagh) [inline PROJECT.md]: popssible

Comment (trasher) [reply PROJECT.md]: Fixed, thanks

File: PROJECT.md [hunk 2]
```diff
@@ -46,4 +52,75 @@ If you provide an array into the `usage` key; its keys will become the propertie
 
 The actual database was designed using GLPI fields; and names are hardcoded. So, if you change `usage` keys; you'll have to provide a mapping beetween your key and the existing one in the databasebase. This is the goal of the `mapping` entry in the configuration.
 
-When working on schema configuration, make sure to tunr debug to true for the application; generated schema would be cached otherwise, and your changes will not be visible :)
+When working on schema configuration, make sure to turn debug to true for the application; generated schema would be cached otherwise, and your changes will not be visible :)
+
+Finally, note tah the `enable_contact` set to false will entirely disable the embed contact page.
```
Comment (orthagh) [inline PROJECT.md]: tah

File: app/Controllers/Contact.php [hunk 3]
```diff
@@ -32,6 +32,6 @@ public function send(Request $req, Response $res)
         );
 
        //redirect
-        return $res->withRedirect('./ok');
+        return $res->withRedirect($this->container->router->pathFor('telemetry'));
```
Comment (orthagh) [inline app/Controllers/Contact.php]: ok to telemetry, is this wanted ?

File: app/Controllers/Reference.php [hunk 4]
```diff
@@ -87,6 +87,6 @@ public function register(Request $req, Response $res)
         );
 
        // redirect to ok page
-        return $res->withRedirect('./ok');
+        return $res->withRedirect($this->container->router->pathFor('reference'));
```
Comment (orthagh) [inline app/Controllers/Reference.php]: same for ok

**Comment by trasher** [comment]
Flash messages are designed to be displayed on any page after a redirection. Having a specific page just to display a message seem  useless to me; a success message on the return page should be enough. But maybe is there something I've missed?

Maybe is there something wrong using flash messages (are they really displayed on every page?); initial fix was only replace the "/ok" call with a named route one.

**Comment by orthagh** [comment]
atm, no confirm messages are displayed

**Comment by trasher** [comment]
Yep, I'll take a look to fix that

**Comment by trasher** [comment]
Looks like this is a JS issue :/

When sending a message with current PR; generate page contains:
```javascript
         $.notify({
      icon: getAlertIcon('success'),
      message: 'Thanks for your message, please wait a bit for our answer !'
   },{
      // settings
      type: 'success',
      delay: getAlertDelay('success'),
      placement: {
         from: 'bottom'
      }
   });
``` 

That sounds correct. Also, bootstrap notify JS file is loaded; but there is the following error: ` $.notify is not a function` :(

**Comment by trasher** [comment]
Since I cannot understand what's wrong with the JS... I've dropped and replaced it with plain HTML that currently do display messages :)

**Pull request merged**